.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Vagrant
+.vagrant
```

Comment:
add blank link at end of file.

Reply:
Added blank line at end of file. waiting for a review.